ipmininet/install/build_vm.sh

```diff
@@ -37,7 +37,7 @@ sudo apt remove -yq python3-pip
 
 [ ! -d "$IPMN_DIR" ] && git clone ${IPMN_REPO} ${IPMN_DIR}
```

Comment:
```suggestion
[ ! -d "$IPMN_DIR" ] && git clone ${IPMN_REPO} ${IPMN_DIR} --branch ${IPMN_BRANCH}
```

Reply:
@oliviertilmans Please review the commit I added 6d84c250aebd161717315b86aa27dc256b79cb9c.